src/test/java/org/mitre/inferno/EndpointTest.java

```diff
@@ -0,0 +1,219 @@
+package org.mitre.inferno;
+
+import org.mitre.inferno.App;
+import org.mitre.inferno.Validator;
+import org.mitre.inferno.rest.IgResponse;
+
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.BufferedReader;
+import org.apache.commons.io.IOUtils;
+import java.net.URL;
+import java.util.List;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+import org.junit.jupiter.api.BeforeAll;
+import static org.junit.jupiter.api.Assertions.fail;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import org.junit.jupiter.api.Test;
+import io.restassured.RestAssured;
+import io.restassured.http.Method;
+import io.restassured.response.Response;
+import io.restassured.response.ResponseBody;
+import io.restassured.specification.RequestSpecification;
+import java.net.URI;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Represents a tester for api endpoints.
+ */
+public class EndpointTest {
```

Comment:
```suggestion
public class ValidatorEndpointTest {
```

Reply:
Probably also want to rename the file to match - renaming to be as specific as possible